lib/reed_solomon_decode_bb_impl.cc

```diff
@@ -115,7 +115,7 @@ namespace gr {
     void
     reed_solomon_decode_bb_impl::forecast(int noutput_items, gr_vector_int &ninput_items_required)
     {
-      ninput_items_required[0] = noutput_items;
+      ninput_items_required[0] = noutput_items*120/110;
```

Comment:
Are you aware that using ints leads to truncation? Basically, 1*120/110 = 1 in this context.